kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue

```diff
@@ -1002,7 +1002,7 @@
       },
       allowLearnersDownloadDescription: {
         message:
-          'Allow users to mark resources for Kolibri to download when available in their network.',
+          'Allow users to explore resources they don’t have and mark it for Kolibri to automatically download when it’s available in their network.',
```

Comment:
Wrong apostrophe character here? Did we agree on `'`?

Reply:
I believe we did!

Reply:
I will be doing a find and replace for all. These are coming from any strings that are copy/pasted from ditto and figma. So, we have some in these new strings (I should have been more attentive today as I was doing this) and some existing strings as well.